src/v/model/tests/random_batch.cc

```diff
@@ -252,6 +253,9 @@ make_random_batches(record_batch_spec spec) {
         o = b.last_offset() + model::offset(1);
         b.set_term(model::term_id(0));
         ret.push_back(std::move(b));
+        if (i % 5 == 0) {
+            co_await ss::coroutine::maybe_yield();
+        }
```

Comment:
should we also yield in the other overload of `make_random_batches` ?

Reply:
done.